File: implement.java
//implement
import java.util.Scanner;   
public class implement {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter your name: ");
        String studentName = scanner.nextLine();
        System.out.print("Enter the number of registered courses: ");
        int numberOfRegisteredCourses = scanner.nextInt();
        
        if (numberOfRegisteredCourses < 7) {
            System.out.println("Overload approval required.");
        } else {
            System.out.println("Registration accepted.");
        }
        
        scanner.close();
    }
}

File: implement.c
// implement 
#include <stdio.h>
int main() {
    char student_name[50];
    int number_of_registered_courses;

    printf("Enter your name: ");
    fgets(student_name, sizeof(student_name), stdin);

    printf("Enter the number of registered courses: ");
    scanf("%d", &number_of_registered_courses);

    if (number_of_registered_courses > 7) {
        printf("Overload approval required.\n");
    } else {
        printf("Registration accepted.\n");
    }

    return 0;
}